### ISSUE TITLE
Release v0.0.11

### ISSUE THREAD

**Issue by vishaltelangre** [libxmljs/libxmljs-dom#6]
Latest version available on https://www.npmjs.com/package/libxmljs-dom is 0.0.10, whereas latest `osmosis` NPM package which depends on this package requires v0.0.11 (as per https://github.com/rchipka/node-osmosis/commit/2ac8b8dec8b88ac017066814142ffe76d96b9f5a).

This is causing following issue.

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for libxmljs-dom@~0.0.11
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'osmosis'
npm ERR! notarget
```

**Comment by rchipka** [comment]
@vishaltelangre Sorry about that. Published v0.0.11.

**Comment by vishaltelangre** [comment]
Thank you.
